src/components/Pages/ManageWiki/Features/PropertiesTable.vue

```diff
@@ -168,24 +182,7 @@ export default {
     }
   },
 
-  created () {
-    this.initialize()
-  },
-
   methods: {
-    initialize () {
```

Comment:
hmm I think this should stay, because the mocks show suggesting properties to the user in this way.
So if there are no equiv entities settings already, we show these suggestions.

Reply:
Oh, nvm https://github.com/wbstack/ui/pull/67 does that.

Reply:
I uploaded #67 where they're initialized in the store instead